Lombiq.VueJs.Samples/Assets/Apps/demo/components/demo-component.js

```diff
@@ -1,19 +1,19 @@
-// Here this is a very simple Vue.js component. As you can see this is not a .vue file. The reason for this is that we
-// can keep our templates in shapes so those can be overridden and localized by Orchard Core. The styling for this is
-// also separated from the Vue app as mentioned before.
-
-import options from '../options';
-
-export default {
-    name: 'demo-component',
-    template: '#demo-component',
-
-    data() {
-        return {
-            showText: false,
-            text: options.text
-        };
-    }
-};
-
+// Here this is a very simple Vue.js component. As you can see this is not a .vue file. The reason for this is that we
+// can keep our templates in shapes so those can be overridden and localized by Orchard Core. The styling for this is
+// also separated from the Vue app as mentioned before.
+
+import options from '../options';
+
+export default {
+    name: 'demo-component',
+    template: '#demo-component',
+
+    data() {
+        return {
+            showText: false,
+            text: options.text
+        };
+    }
+};
+
 // NEXT STATION: Views/VueComponents/Demo.Component.cshtml
```

Comment:
I can't find the guideline right now, but I recall that we should add a line break at the end of each file. There is one in _demo/main.js_ but not here, and also not in some other files.

It seems like a good occasion to consolidate this.

Reply:
I searched through with regex and added the line break to code files, excluding the auto generated.